core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -1401,8 +1532,11 @@ class SparkContext(config: SparkConf) extends Logging {
 
   /**
    * Broadcast a read-only variable to the cluster, returning a
-   * [[org.apache.spark.broadcast.Broadcast]] object for reading it in distributed functions.
+   * `org.apache.spark.broadcast.Broadcast` object for reading it in distributed functions.
```

Comment:
Yes, the initial intent was to make it the same everywhere (either backquotes or brackets)